src/test/java/net/datafaker/providers/base/AddressTest.java

```diff
@@ -273,4 +273,11 @@ void dutchAddress() {
     void australiaAddress() {
         assertThat(AU_FAKER.address().fullAddress()).matches("(Unit|[0-9]).+, [A-Z].+, [A-Z]{2,3} [0-9]{4}");
     }
+
+    @RepeatedTest(100)
+    void cnAddressTest() {
```

Comment:
This isn't testing address, it's testing suffix

Reply:
ok ，I deleted this code and remove the unrelated commit.。